ai-assistant-product/README_DE.md

```diff
@@ -1 +1,667 @@
+
+```json
+{
+    "stepNo": 1,
+    "type": "IVY_TOOL",
+    "toolId": "find-web-developer",
+    "onSuccess": 3,
+    "onError": 2,
+    "customInstruction": "Find employees has role 'WEB_DEVELOPER'.",
+    "saveToHistory": false
+}
+```
+
+##### Text-Schritt
+
+Der **Text-Schritt** ist eine zentrale Komponente in KI-Workflows, die darauf ausgelegt ist, textbasierte Inhalte für die Benutzerinteraktion anzuzeigen oder zu generieren. Abhängig von seiner Konfiguration kann der Text-Schritt feste Nachrichten, KI-generierte Inhalte, Ergebnisse vorheriger Schritte oder sogar versteckte Nachrichten für die interne KI-Verarbeitung anzeigen. Diese Flexibilität ermöglicht es der KI, effektiv mit den Benutzern zu kommunizieren und gleichzeitig Entscheidungsprozesse zu steuern.
+
+**Fester Text**
+
+- **text**: Der feste Text, der in der Benutzeroberfläche angezeigt werden soll.
+
+- **showResultOfStep**: Option, das Ergebnis eines vorherigen Schritts anzuzeigen, indem auf dessen Nummer verwiesen wird.
+
+- **onSuccess**: Definiert den nächsten Schritt, wenn der Benutzer nach dem Lesen des Textes eine Eingabe macht.
+
+```json
+{
+    "stepNo": 2,
+    "type": "TEXT",
+    "text": "I have rephrased your request as follows. Could you please confirm if it is correct?",
+    "showResultOfStep": 0,
+    "onSuccess": 3
+}
+```
+
+**KI-generiert**
+
+- **useAI**: Auf „true“ setzen, um der KI zu erlauben, Inhalte dynamisch zu generieren, wie beispielsweise Zusammenfassungen oder Berichte.
+
+- **customInstruction**: Eine Leit-Anweisung, um der KI zu helfen, passenden Text basierend auf dem Kontext oder den Benutzereingaben zu erstellen.
+
+- **onSuccess**: Der nächste Schritt, der nach der Benutzerinteraktion ausgeführt werden soll.
+
+```json
+{
+    "stepNo": 1,
+    "type": "TEXT",
+    "useAI": true,
+    "customInstruction": "Use the conversation above to summarize information of the planned project in a structured format. If user didn't provide a name for the project generete the project name based on description of the project. Example: '**Project name:** ProjectA\n**Project description:** description of projectA\n**Technologies:** tech stack\n**Members:** show member information'. Then add a line to ask if need some update",
+    "onSuccess": 2
+},
+```
+
+**Ergebnis eines anderen Schritts anzeigen**
+
+- **showResultOfStep**: Zeigt das Ergebnis eines zuvor ausgeführten Schritts an.
+
+```json
+{
+    "stepNo": 1,
+    "type": "IVY_TOOL",
+    "toolId": "find-employees",
+    "onSuccess": 3,
+    "onError": 2,
+    "customInstruction" : "Find employees has the tech stack are the technologies above.",
+    "useConversationMemory": true,
+    "saveToHistory": false
+},
+{
+    "stepNo": 3,
+    "type": "TEXT",
+    "text": "I found these suitable employees matched the requirement. Do you want to choose some of them for the project?",
+    "showResultOfStep" : 1,
+    "useConversationMemory": false,
+    "onSuccess": 4
+}
+```
+
+**Versteckt**
```

Comment:
do not translate names of variables

Reply:
Sorry, I mean "hidden Text Step", hide the text step from user, only AI can read it.

translate to German: "versteckter Text Schritt"